OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj

```diff
@@ -43,16 +43,16 @@
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="OpenRA.Game">
-      <HintPath>dependencies\OpenRA.Game.exe</HintPath>
+      <HintPath>..\engine\OpenRA.Game.exe</HintPath>
```

Comment:
If you change these to project references then you'll be able to use the "Jump to Definition" IDE feature.